radis/io/hitemp.py

```diff
@@ -154,20 +150,40 @@ def fetch_hitemp(
         databank_name = databank_name.format(**{"molecule": molecule})
     local_databases = abspath(local_databases.replace("~", expanduser("~")))
 
+    if databank_name in getDatabankList():
```

Comment:
GetDatabankList creates an error if the Radis.json has never been created yet (ie : first run of Radis)

Reply:
In the current version the database is not re-downloaded if there already ; i think it happens in loader.py / Factory.fetch_hitemp.  Did it not work in the CO2/H2O case and you had to add this test?

Reply:
@dcmvdbekerom I may have a look tomorrow; if you can answer this !

Reply:
There used to be a test after the download that checks if the database exists, and if so, throw an exception. This resulted in having to re-download the entire thing again. For H2O that is over an hour, so I wanted to check before the download so that it will be processed without issues once the download starts. But perhaps the check currently is too early.

Reply:
Right ! I think thats because Datasource downloads its own cache files, keeps them, and reuses them if required. Therefore it wouldn't re-download the second time. 

For the zip files of CO2 / H2O you may not have these cache files.